11ty/definitions/nl_NL/biromantic.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Biromantic
+slug: biromantic
+speech: adj
+defined: true
+reading:
+  - text: Sorting Out Romantic vs Sexual Feelings
+    href: http://teenhealthsource.com/blog/faq-sorting-out-romantic-vs-sexual-feelings/
+  - text: Being Bi On The Asexual Spectrum
+    href: https://www.curvemag.com/Lifestyle/Being-Bi-On-The-Asexual-Spectrum-2141/
+  - text: Here’s What It Means When Your Romantic and Sexual Orientations Are Different
+    href: https://everydayfeminism.com/2016/07/cross-orientation-101/
+---
+
+Romantisch aangetrokken zijn tot meer dan één geslacht of sekse.
+
+## Noot
+
+Biromantic impliceert geen seksuele aantrekkingskracht. Een biromantic persoon kan aseksueel, homoseksueel, heteroseksueel, biseksueel, pan seksueel etc. zijn.
```

Comment:
Would it be "biromantisch" in Dutch? Or is it being kept as the English phrase intentionally?

Would "panseksueel" be one word like the others?

Reply:
A Dutch variant is not quite available, that's why I opted for keeping the English phrase.